### PR TITLE
Fix inconsistent return types in pybind11 function prototypes

### PR DESCRIPTION
**Pull request by mattdawkins**
This was required for me to build on windows on VS17/19, but several function prototypes in the pybind11 file did not match those in neighboring files due to differences in returning ints vs voids, and one const inconsistency on recent code

### CODE REVIEW

File: mmcv/ops/csrc/pytorch/carafe_naive.cpp [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #include "pytorch_cpp_helper.hpp"
 
 #ifdef MMCV_WITH_CUDA
-void CARAFENAIVEForwardCUDAKernelLauncher(const Tensor features,
-                                          const Tensor masks, Tensor output,
-                                          const int kernel_size,
-                                          const int group_size,
-                                          const int scale_factor);
+int CARAFENAIVEForwardCUDAKernelLauncher(const Tensor features,
```
Comment (xvjiarui) [inline mmcv/ops/csrc/pytorch/carafe_naive.cpp:4]: I suggest removing `return` in `CARAFENAIVEForwardCUDAKernelLauncher`.

**Comment by xvjiarui** [comment]
Hi @mattdawkins 
Could you please sign CLA? 
And there is some linting error, could you please fix it?

**Comment by mattdawkins** [comment]
I don't think I removed any input arguments themselves, just the const on the tin function args (to make it more consistent with the others and that was throwing another VS error)

File: mmcv/ops/csrc/pytorch/pybind.cpp [hunk 2]
```diff
@@ -3,43 +3,43 @@
 std::string get_compiler_version();
 std::string get_compiling_cuda_version();
 
-int carafe_naive_forward(Tensor features, Tensor masks, Tensor output,
-                         int kernel_size, int group_size, int scale_factor);
-
-int carafe_naive_backward(Tensor top_grad, Tensor features, Tensor masks,
-                          Tensor bottom_grad, Tensor mask_grad, int kernel_size,
-                          int group_size, int scale_factor);
```
Comment (xvjiarui) [inline mmcv/ops/csrc/pytorch/pybind.cpp:11]: `kernel_size` should not be removed.

File: mmcv/ops/csrc/pytorch/pybind.cpp [hunk 3]
```diff
@@ -3,43 +3,43 @@
...
-int deform_conv_backward_input(Tensor input, Tensor offset, Tensor gradOutput,
-                               Tensor gradInput, Tensor gradOffset,
-                               Tensor weight, Tensor columns, int kW, int kH,
-                               int dW, int dH, int padW, int padH,
-                               int dilationW, int dilationH, int group,
-                               int deformable_group, int im2col_step);
-
-int deform_conv_backward_parameters(Tensor input, Tensor offset,
-                                    Tensor gradOutput, Tensor gradWeight,
-                                    Tensor columns, Tensor ones, int kW, int kH,
```
Comment (xvjiarui) [inline mmcv/ops/csrc/pytorch/pybind.cpp:38]: `kH` should not be removed.

**Comment by xvjiarui** [comment]
Hi @mattdawkins 
Sorry. It is the display issue of GitHub. 
LGTM

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**